browserlessio.json

```diff
@@ -14,9 +14,9 @@
     "utctime_updated": "2021-11-10T04:57:59.000000Z",
     "package_name": "phantom_browserlessio",
     "main_module": "browserlessio_connector.pyc",
-    "min_phantom_version": "4.8.24304",
+    "min_phantom_version": "5.0.0",
```

Comment:
IRC we want to avoid changing the min phantom version with fips compliant change. Could you confirm if this is intentional?

Reply:
I would have to find the post, but this is referring to when the change is only in the app.json. The change I have included (already merged to next), involves an actual code change therefore I updated the min version. My original PR did not so that's why this new one exists